syndeo-template/src/main/java/com/google/cloud/syndeo/common/ProviderUtil.java

```diff
@@ -117,11 +117,19 @@ public static PCollectionRowTuple applyConfigs(
       Collection<TransformSpec> specs, PCollectionRowTuple tuple) {
     for (TransformSpec spec : specs) {
       SchemaTransform transform = spec.provider.from(spec.configuration);
+
+      if (tuple.getAll().containsKey("errors")) {
```

Comment:
Is my understanding correct that we would be routing the "errors" to the dlq once it is ready?

Reply:
that's correct!